tests/integrationTests/inputs/radiation/spectrumRadiation.yaml

```diff
@@ -0,0 +1,89 @@
+#
```

Comment:
Can you add some documentation to this example that explains why it is being used and what it does?